packagefile.go

```diff
@@ -353,6 +354,37 @@ func (c *PackageFile) Files() []FileInfo {
 	return c.files
 }
 
+// fileModeFromInt64
```

Comment:
Can you please expand this comment with the context you gave in the PR? Just noting why the cast to `os.FileMode` is not correct.

Reply:
Comment expanded. I have just called the flags by name (e.g. syscall.S_IFBLK and os.ModeDevice) rather than actual links - let me know if you would prefer more. The flags were already used in the function, but I have filled in values in the comment.